biz.aQute.bndlib/src/aQute/bnd/build/ProjectBuilder.java

```diff
@@ -535,7 +535,7 @@ public String toString() {
 	 * version.
 	 */
 	protected void startBuild(Builder builder) {
-		project.versionMap.clear();
+		project.versionMap.remove(builder.getBsn());
```

Comment:
Are these uses of versionMap safe? In some places Project, versionMap is synchronized before being mutated.